src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,250 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import {
+  deleteTodo,
+  getTodos,
+  updateTodoStatus,
+  updateTodoTitle,
+} from '../../api/todos';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+import cn from 'classnames';
+
+type Props = {
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  setErrorMessage: React.Dispatch<React.SetStateAction<string>>;
+  filteredTodos: Todo[];
+  todos: Todo[];
+  tempTodo: Todo | null;
+  setLoadingTodosId: React.Dispatch<React.SetStateAction<number[]>>;
+  loadingTodosId: number[];
+  addIdToLoad: (arg: number) => void;
```

Comment:
addIdToLoad and setLoadingTodosId are almost the same functions, remove one function